composer.json

```diff
@@ -16,17 +16,17 @@
         }
     ],
     "require":      {
-        "php": ">=7.1",
+        "php": ">=7.2.5",
         "knplabs/gaufrette": "^0.11",
-        "symfony/config": "^4.4|^5.0|^6.0",
-        "symfony/dependency-injection": "^4.4|^5.0|^6.0",
-        "symfony/http-kernel": "^4.4|^5.0|^6.0"
+        "symfony/config": "^5.0|^6.0|^7.0",
+        "symfony/dependency-injection": "^5.0|^6.0|^7.0",
+        "symfony/http-kernel": "^5.0|^6.0|^7.0"
     },
-    "require-dev": {
-        "symfony/phpunit-bridge": "^5.4",
-        "symfony/console": "^4.4|^5.0|^6.0",
-        "symfony/filesystem": "^4.0|^5.0|^6.0",
-        "symfony/yaml": "^4.0|^5.0|^6.0"
+    "require-dev":  {
+        "symfony/phpunit-bridge": "^5.4|^7.0",
```

Comment:
let's drop version 5 for the bridge, the latest version is meant to be used with all Symfony versions